main/zoe/api/zoe.md

```diff
@@ -200,14 +245,133 @@ either the `proposal `or `payments` are empty, indicate this by
+Zoe and the contract. The party who exercises an `invitation` and sends the `offer()` message
+to Zoe gets a `UserSeat` that can check `payouts`' status or retrieve their `offer` result.
+The result is whatever the contract chooses to return. This varies, but examples
+are a `string` and an `invitation` for another user.
+
+Also, a `userSeat` can be handed to an agent outside Zoe and the contract, letting
+them query or monitor the current state, access the `payouts` and result,
+and, if it's allowed for this `seat`, call `tryExit()`. 
+
+Since you can potentially exit the `seat` if you have a reference to it, you should
+only share your `UserSeat` with very trusted parties.
+
+A `UserSeat` has eight methods, six of which are 'get' methods. Of these six,
+three (`getCurrentAllocation()`, `hasExited()`, `getNotifier()`) return values
+from the `seat`'s current state, one (`getProposal()`) returns a value from an
+unchanging state (i.e. You can't change the `proposal` associated with a `seat`),
+and three (`getPayouts()`, `getPayout()`, `getOfferResult()`) extract a result.
+
+Another is 
+a `boolean` returning test, and the last attempts an action. Note that `tryExit()` 
+only works if the `seat`'s `proposal` has `OnDemand` for its `exit` clause.  
+
+- `getCurrentAllocation()`
+  - Returns: `{ Promise<Allocation> }`
+  - An `Allocation` is an `AmountKeywordRecord` of key-value pairs where
+    the key is a keyword such as `Asset` or `Price` applicable to the
+    contract. The value is an `amount` with its `value` and `brand`. 
+    
+    `Allocations` represent the `amounts` to be paid out to each `seat` on exit. 
+    Possible exits are exercising an exit condition, the contract's explicit choice, 
+    or a crash or freeze. There are several methods for finding out what `amount` 
+    a current `allocation` is.
+    
+    An `Allocation` example:
+    - ```js
+      {
+        Asset: quatloosAmountMath.make(5),
+        Price: moolaAmountMath.make(9)
+      }
+      ```
+- `getProposal()`
+  - Returns: `{ Promise<ProposalRecord> }`
+  - A `Proposal` is represented by a `ProposalRecord`. It is the rules
+    accompanying the escrow of `payments` dictating what the user expects
+    to get back from Zoe. It has keys `give`, `want`, and
+   `exit`. `give` and `want` are records with keywords as keys and
+    `amounts` as values. The proposal is a user's understanding of the
+    contract that they are entering when they make an offer. See
+    `E(zoe).offer()` for full details.  
+  - Example:
+    ```js
+    const { want, give, exit } = sellerSeat.getProposal();
+    ```
+- `getPayouts()`
+  - Returns: `{ Promise<PaymentPKeywordRecord> }`
+  - A `payout` is a `payment` that goes to a party in a successful transaction, redirecting
+    escrowed assets in accordance with the result of the transaction. Returns a record
+    of all the `payout` `payments` associated with the `seat`'s offers.
```

Comment:
sorry, but "Returns a record of all the payouts" sounds a little like a receipt. In this case it contains actual payments.
```suggestion
    escrowed assets in accordance with the result of the transaction. Returns a record
    containing all the `payout` `payments` associated with the `seat`'s offers.
```